recipes/coin-clp/all/conanfile.py

```diff
@@ -57,8 +57,12 @@ def configure(self):
             del self.options.fPIC
 
     def requirements(self):
-        self.requires("coin-utils/2.11.4")
-        self.requires("coin-osi/0.108.6")
+        if self.version == "1.17.6":
+            self.requires("coin-utils/2.11.4")
+            self.requires("coin-osi/0.108.6")
+        else:
+            self.requires("coin-utils/2.11.6")
+            self.requires("coin-osi/0.108.7")
```

Comment:
Can't you just require latest version? I can't believe there are breaking changes in patch versions.

Reply:
Plus one, I'd love to know why this is conditional :)

Reply:
I agree there is technically no reason.
It is mostly due to numerical stability in results : when consuming cbc version x from CCI, or building using coin-or `coinbrew` script, one can expect to have the same results, whose values depend on the whole "coin library chain" (cbc / cgl / clp / osi / utils). Thus the "need" to force versions.
Not blocking though.

Reply:
There's a separation in Conan with the "source code version" and the "how the binaries are built version" .. the later is called recipe revisions and it's required for having the same results.

There's docs/how_to_consume if I recall for the details and helpful links you need to do that